stable/mission-control/values.yaml

```diff
@@ -99,7 +58,7 @@ postgresql:
   enabled: true
   imageTag: "9.6.11"
   postgresUsername: postgres
-  postgresPassword:
+  postgresPassword: postgres
```

Comment:
We should not hardcode a default password here. Let it be auto generated or passed as a parameter or a secret. 
Also need to document that in case of allowing auto generation, the following `helm upgrade` command must include this password so not to generate a new one.

Reply:
+1